src/services/y-websocket.js

```diff
@@ -0,0 +1,539 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
The file license is probably still MIT then, but not entirely sure, probably something to clarify

I didin't diff how much it is based on it (https://github.com/yjs/y-websocket/blob/master/LICENSE)